Snakefile

```diff
@@ -32,14 +32,29 @@ rule fastq_dump:
         fastq-dump -O $(dirname {output.R1}) --split-files --gzip {wildcards.SRR_ID}
         """
 
+rule fastqc:
+    input:
+        R1 = int_dir / "fastqs" / "{SRR_ID}" / "{SRR_ID}_1.fastq.gz",
+        R2 = int_dir / "fastqs" / "{SRR_ID}" / "{SRR_ID}_2.fastq.gz",
+    output:
+        fastqc_1 = int_dir / "fastqc" / "{SRR_ID}" / "{SRR_ID}_1.fastq" /,
```

Comment:
My understanding of fastqc is that its output is HTML. Does this not fail when it can't find the output?